### ISSUE TITLE
Strangeness doesn't have a manifestation text.

### ISSUE THREAD

**Issue by Durkel** [tgstation/-tg-station#7764]
This is a absolute mega pain in the balls and confuses geneticists to no end. Several times Coworkers and Myself have been stumped by the puzzling fact that shit just keeps mutating 4noraisin. Please give it a text confirmation.

**Comment by dannno** [comment]
I'm fairly certain it does and you just mutate if you have high rads/ overdose on SE's even if you don't have it.

**Comment by Durkel** [comment]
I've had this happen over three games and it's the only logical explanation I've been able to come up. I don't recall seeing the strangeness text. As I said too, I waited until the radiation levels are zero, and the effect would still occur.

**Comment by WJohn** [comment]
@razharas or @iamgoofball

**Comment by Iamgoofball** [comment]
if you're talking about Bad DNA then it has a manifestation text. It's "You feel strange."

**Comment by Iamgoofball** [comment]
it's also entirely possible you had the DNA Aide virus symptom somehow

**Comment by Durkel** [comment]
I've checked for the virus the three separate times I've encountered this. I'll add that I've only had this problem occur 30+ minutes in the round, normally around block 15-20, depending on how fast I'm able to work. I wasn't getting a text for strangeness on myself with a injection, but the pulsing the SE in the scanner just the lightest amount with zero radiation was causing radical block movements. I've even admin helped to see if a coder online could help, but so far they've got nothing. It could possible be something besides strangeness, but that's what me and someone else were able to come up with.
